source/includes/_payment.md

```diff
@@ -936,14 +1023,20 @@ curl -v \
 import org.killbill.billing.client.api.gen.PaymentApi;
 protected PaymentApi paymentApi;
 
+UUID payment_id = UUID.fromString("b9976ff0-d326-42bd-8e91-ea083149a38f");
+
 PaymentTransaction body = new PaymentTransaction();
 body.setPaymentExternalKey("a975f7b8-1e59-4801-91c9-fcce1526b019");
 body.setAmount(new BigDecimal(5));
 		
 List<String> NULL_PLUGIN_NAMES = null;
 Map<String, String> NULL_PLUGIN_PROPERTIES = null;
 		
-Payment payment = paymentApi.chargebackPaymentByExternalKey(body, NULL_PLUGIN_NAMES, NULL_PLUGIN_PROPERTIES, requestOptions);
+Payment payment = paymentApi.chargebackReversalPayment(payment_id, 
+                                                       body, 
```

Comment:
Is this code alignment correct?

Reply:
Hi Reshma, yup its correct.

![image](https://github.com/killbill/slate/assets/75751774/15b1b44f-548c-4321-8362-4b64d8b3f011)